src/Model/Lotto.js

```diff
@@ -0,0 +1,35 @@
+import { validateLotto } from '../Validator/Validate.js';
+class Lotto {
+  #numbers;
+
+  constructor(numbers) {
+    this.#validate(numbers);
+    this.#numbers = numbers;
+  }
+
+  getNumbers() {
+    return this.#numbers.sort((a, b) => a - b);
+  }
+
+  #validate(numbers) {
+    const [check, message] = validateLotto(numbers.sort((a, b) => a - b));
+    if (check) {
+      throw new Error(message);
+    }
+  }
+
+  calculateCorrectNumber(lottoArray, bonusNumber) {
+    const correctArray = [];
+    lottoArray.forEach((lotto) => {
+      correctArray.push(this.#compareLottoAndGoal(lotto, Number(bonusNumber)));
+    });
```

Comment:
```suggestion
    const correctArray = lottoArray.map((lotto) => this.#compareLottoAndGoal(lotto, Number(bonusNumber));

```
이런 식의 표현은 어떠신가요? 🙂 push는 기존 배열을 수정하는 건데, 이런 불변성을 깨는 액션은 최대한 적게 하는 게 좋다고 하더라구요! map, reduce, filter 등의 메소드를 활용하시면 기존 배열이 변화되지 않고 새 배열이 반환되어 코드가 조금 더 깨끗해지더라구요

불변성 유지에 관한 글을 찾아서 첨부드립니다~! 시간 되실 때 한 번 읽어보셔도 좋을 것 같아요
https://evan-moon.github.io/2020/01/05/what-is-immutable/